src/command/src.jl

```diff
@@ -0,0 +1,32 @@
+function commander(c::Client, m::Message, ::Val{:source})
```

Comment:
Can you please reformat all code with 4 spaces for indenting?  Also remove all tabs. We mostly follow the [Blue Style Guide](https://github.com/invenia/BlueStyle#synopsis).

Same for main.jl and constants.jl